src/lib/Behat/PageElement/Notification.php

```diff
@@ -21,6 +22,7 @@ public function __construct(UtilityContext $context)
             'alert' => '.alert',
             'successAlert' => '.alert-success',
             'failureAlert' => '.alert-danger',
+            'closeAlert' => '.alert-success.show button',
```

Comment:
I have some suggestions to make it more bulletproof. 
`closeAlert` do not match to the class `.alert-success` - this should be `closeSuccessAlert` or use selector '.alert'.
The `button` is to generic, there might be at some point more buttons in alert, I would rather use `button.close` selector.